test/features/navigation.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Navigation
+
+    Scenario: Check for the presence of the "Request data" link in header when visiting as a non-logged in user
+        When I go to homepage
+        # Make the comparison case-insensitive
+        Then I should see an element with xpath "//a[contains(translate(., 'RD', 'rd'), "request data")]"
+
+    Scenario: Check for the presence of the "Request data" link in header when visiting as a logged in user
+        Given "Admin" as the persona
+        When I log in
+        Then I go to homepage
+        # Make the comparison case-insensitive
+        Then I should see an element with xpath "//a[contains(translate(., 'RD', 'rd'), "request data")]"
+
+    Scenario: Check for the presence of the "Copyright" link in footer when visiting as a non-logged in user
+        When I go to homepage
```

Comment:
Why no given?

as all of these tests start at the homepage? 

use background: to prep the system to be on the home page.
https://sukesh15.gitbooks.io/cucumber-jvm-test-framework-/content/cucumber_-_more_details/understanding_background_in_cucumber.html